tests/partition/evolutionary/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_gmock_test(combine_test combine_test.cc)
+target_link_libraries(combine_test ${Boost_LIBRARIES})
+add_gmock_test(population_test population_test.cc)
+target_link_libraries(population_test ${Boost_LIBRARIES})
+add_gmock_test(individual_test individual_test.cc)
+add_gmock_test(probability_table_test probability_table_test.cc)
+add_gmock_test(edge_frequency_test edge_frequency_test.cc)
+add_gmock_test(mutation_test mutation_test.cc)
+target_link_libraries(mutation_test ${Boost_LIBRARIES})
+add_gmock_test(evo_partitioner_test evo_partitioner_test.cc)
+target_link_libraries(evo_partitioner_test ${Boost_LIBRARIES})
```

Comment:
IBM09 might be too large for tests in debug mode. why didn't you use ibm01, which is already contained in the framework?